opengrok-tools/src/main/python/opengrok_tools/utils/log.py

```diff
@@ -0,0 +1,79 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# See LICENSE.txt included in this distribution for the specific
+# language governing permissions and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at LICENSE.txt.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+#
+
+import logging
+import sys
+
+
+class MinLogLevelFilter(logging.Filter):
+    def __init__(self, level):
+        super()
+        self.level = level
+
+    def filter(self, record):
+        if record.levelno >= self.level:
+            return True
+
+        return False
+
+
+class MaxLogLevelFilter(logging.Filter):
+    def __init__(self, level):
+        super()
+        self.level = level
+
+    def filter(self, record):
+        if record.levelno <= self.level:
+            return True
+
+        return False
+
+
+def get_console_logger(name, level=logging.INFO):
+    """
+    Get logger that logs logging.ERROR and higher to stderr, the rest
+    to stdout.
+    :param level: base logging level
+    :param name: name of the logger
+    :return: logger
+    """
+    formatter = logging.Formatter('%(message)s')
```

Comment:
this will change also all other log messages to be in this format (in debug mode - for urllib and so on), is it intended?

Reply:
With `propagate = False` it should not affect any other logger, including the root one.